src/main/resources/application.properties

```diff
@@ -16,3 +16,4 @@ spring.flyway.placeholders.schema_name = ${SCHEMA_NAME}
 
 cors.origin.allowed=${CORS_ORIGIN_ALLOWED}
 minutes.ofTollerance.multiplier=${MIN_OF_TOLLERANCE_MULTIPLIER}
```

Comment:
just FYI, this config corresponds to 
```
minutes {
  ofTollerance {
    multiplier=${MIN_OF_TOLLERANCE_MULTIPLIER}
  }
}
```
which in terms of settings group is not correct.

the right naming would be `minutesOfTolleranceMultiplier`, or even better `tolleranceMultiplierInMinutes`